kyo-scheduler/jvm/src/main/scala/kyo/scheduler/regulator/Regulator.scala

```diff
@@ -43,7 +41,7 @@ abstract class Regulator(
 
     protected def measure(v: Long): Unit = {
         probesCompleted.increment()
-        stats.measurement.observe(v.toDouble)
+        stats.measurement.observe(Math.max(0, v.toDouble))
```

Comment:
Workaround because HdrHistogram doesn't support negative values and there's a test that reproduces negative measurements.